src/api/app/services/notification_creator.rb

```diff
@@ -0,0 +1,27 @@
+class NotificationCreator
+  EVENTS_TO_NOTIFY = ['Event::RequestStatechange',
+                      'Event::RequestCreate',
+                      'Event::ReviewWanted',
+                      'Event::CommentForProject',
+                      'Event::CommentForPackage',
+                      'Event::CommentForRequest'].freeze
+
+  def initialize(event)
+    @event = event
+  end
+
+  def call
+    return unless @event.eventtype.in?(EVENTS_TO_NOTIFY)
+    @event.subscriptions.each { |subscription| create_notification_per_subscription(subscription) }
+  rescue StandardError => e
+    Airbrake.notify(e, event_id: @event.id)
```

Comment:
Same here. what is the difference to let the exception through?

Reply:
Let's talk about this.

Reply:
We talked and came to the conclusion to leave this alone for now